sympy/physics/control/tests/test_lti.py

```diff
@@ -1811,6 +1811,37 @@ def test_StateSpace_functions():
     assert ss3.feedforward_matrix == Matrix([[0, 0], [0, 1]])
 
 
+def test_StateSpace_symbolics():
+    # Test for using StateSpace with symbolic matrices.
+    a11, a12, a21, a22, b1, b2, c1, c2 = symbols('a_11 a_12 a_21 a_22 b_1 b_2 c_1 c_2')
```

Comment:
No need to declare symbols separately here.
Some of these are already present here - https://github.com/sympy/sympy/blob/452bd0011ed85a2bfeae8edc5f652caade6ec0e5/sympy/physics/control/tests/test_lti.py#L24-L27